### ISSUE TITLE
how to put a quote into yaml?

### ISSUE THREAD

**Issue by dankelley** [codecov/support#318]

**Comment by dankelley** [comment]
At the end of my `.travis.yml` for an R-language project, I have
```yaml
after_success:
        - Rscript -e 'library(covr); codecov(type=''all'')'
```
The travisCI build finished 12 hours ago (https://travis-ci.org/dankelley/oce) but the codecov test is still for an old version (https://codecov.io/gh/dankelley/oce/commit/01d335ceb1d370e0ed773212c5d1ef7b7f35dfa1). The difference is that, for the old version, I did not have `type=''all''` specified.

Is the problem in my using two single-quote characters (`'`)? I have also tried using double-quote characters (`"`) but that also fails. I don't know whether this is a problem with my yaml syntax, or with the `codecov` call, itself. What I *do* know, however, is that if I run the yaml stated above in a local test (using double quote), then it works fine.

I apologize if this is something I should have learned in reading documentation. I did spend time reading that documentation, and think that what I am doing might be right.. ??
